core/utils/htable.cc

```diff
@@ -376,6 +369,7 @@ void *HTableBase::GetHash(uint32_t pri, const void *key) const {
 
   /* check primary bucket */
   ret = get_from_bucket(pri, pri, key);
+
```

Comment:
Surely it's a realm of personal preference, but can we not have a blank line between an assignment and its failure check? The Guide recommends minimizing the use of vertical space... (although I don't always agree with the principle)